src/containers/permissions/DataSetPermissionsContainer.js

```diff
@@ -162,47 +135,37 @@ const DataSetPermissionsContainer = ({
     }
   };
 
-  const reducedRS :?RequestState = reduceRequestStates([
-    getPermissionsRS,
-    initializePermissionsRS
-  ]);
-  const requestIsPending = isPending(reducedRS);
-  const requestIsSuccess = isSuccess(reducedRS);
-
   return (
     <StackGrid gap={8}>
       {
-        requestIsPending && (
+        isPending(getDataSetPermissionsPageRS) && (
           <Spinner />
         )
       }
       {
-        requestIsSuccess && filteredPermissionsCount === 0 && (
+        isSuccess(getDataSetPermissionsPageRS) && totalPermissions === 0 && (
           <Typography align="center">No datasets.</Typography>
         )
       }
       {
-        requestIsSuccess && filteredPermissionsCount !== 0 && (
-          pageDataSets.valueSeq().map((dataSet :EntitySet | Map) => {
-            const dataSetId :UUID = getDataSetField(dataSet, 'id');
-            return (
-              <DataSetPermissionsCard
-                  dataSet={dataSet}
-                  isOpen={openDataSetId === dataSetId}
-                  key={dataSetId}
-                  onClick={toggleOpenDataSetCard}
-                  onSelect={onSelect}
-                  organizationId={organizationId}
-                  principal={principal}
-                  selection={selection} />
-            );
-          })
+        isSuccess(getDataSetPermissionsPageRS) && totalPermissions !== 0 && (
```

Comment:
Well, `requestIsSuccess` was the result of the reduced request state, which doesn't exist anymore. Separately, regarding your comment about pulling it into a variable, I don't necessary find much value in that. Yes, it avoids the overhead of the function call, but all the function does is an `===` equivalence check, which is fast. Objectively, `isSuccess(getDataSetPermissionsPageRS)` and `getDataSetPermissionsPageIsSuccess` are basically the same amount of characters, so that's a no-op.

Reply:
yeah, it's a personal preference, so you do you!